app/Espo/Services/Record.php

```diff
@@ -2257,8 +2257,9 @@ public function massUpdate($data, array $params)
         $this->filterInput($data);
 
         $params['action'] = 'update';
-        $params['maxActionCount'] =  $this->getConfig()->get('maxMassUpdateCount', 20);
-        $params['chunkSize'] = $this->getConfig()->get('massUpdateChunkSize', 2000);
+        $params['maxCountWithoutJob'] = $this->getConfig()->get('massUpdateMaxCountWithoutJob', 200);
```

Comment:
massUpdateMaxCountWithoutJob, massUpdateMaxChunkSize, massUpdateMinChunkSize needs to be added to default config.